### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by H-Huang**
typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ pip install --pre torch torchvision torchtext torchaudio -f https://download.pyt
 
 Install other necessary libraries:
 ```
-pip install pyyaml
+pip install pyyaml pytest
```
Comment (xuzhao9) [inline README.md:41]: If `distro` is also missing, we should add it here too.

**Review by xuzhao9 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (H-Huang) [reply README.md:41]: Actually I noticed that pytest, distro, etc. are included in requirements.txt. Not sure how come it wasn't installed as part of `python install.py`. I will double check

Comment (H-Huang) [reply README.md:41]: seems like its my fault and the initial install.py didnt succeed for me. Now i see the packages. I'll remove the package update and only check in the typo change

Comment (xuzhao9) [reply README.md]: Sounds good to me, thanks!

**Comment by xuzhao9** [comment]
Hi @H-Huang , can you help Import this diff to fbsource so that we can land it?

**Pull request closed without merging**